node-operators/appchain-node/rpc-systemd.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Run an RPC Node Using Systemd
+description: Learn how to set up and run a Tanssi Appchain node using Systemd, which allows you to have your own RPC endpoint to interact with your Appchain.
+---
+
+# Run an Appchain Node Using Systemd
+
+## Introduction {: #introduction }
+
+--8<-- 'text/node-operators/appchain-node/intro.md'
+
+In this guide, you'll learn how to spin up a Tanssi Appchain node using a binary executable file and manage the service with [Systemd](https://systemd.io/){target=\_blank} on Linux systems.
+
+The article follows the good practice of running the service with its own non-root account and granting that account writing access to a specific directory. However, do adapt this article's steps and instructions to your infrastructure configuration, preferences, and security policies.
+
+!!! note
+    It is not possible to run an RPC node for Snap Appchains as they run on a private network, and their nodes are, therefore, unreachable for syncing.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To get started, you'll need access to a computer running an Ubuntu Linux OS and root privileges. You will also need:
+
+- **Node binary file** - the instructions in this guide execute the [latest](https://github.com/moondance-labs/tanssi/releases/latest){target=\_blank} official stable node release. If you want to build and run your own file, make sure to follow the instructions for [building your Appchain node](/builders/build/customize/prerequisites){target=\_blank}
+
+- **Appchain specifications file** - the Appchain specification file is needed to run the node. You can download it from the dashboard in the [dApp](https://apps.tanssi.network/){target=\_blank} by clicking the `Appchain Data` link
+
+    ![Getting the chain specs](/images/node-operators/appchain-node/rpc-systemd/rpc-systemd-1.webp)
```

Comment:
Just curious - do you know what zoom % you use when taking screenshots? I feel like these screenshots are a bit blurry and I'm not sure if it's like a webp thing or the size of the screenshot vs on the docs

Reply:
I also prefer, if possible, having the browser in the screenshot instead of just part of the screen

Reply:
hmm, I wasn't using any zoom at all 
Maybe low res in the export settings? Now I replaced the image and exported the webp using the same setting (3300x1819 px, 300DPI). it looks good to me, but let me know otherwise!

Reply:
looks great!! it probably doesn't help that i'm viewing it on a laptop screen